src/affiliate-manager/components/admin/affiliates/allAffiliates.js

```diff
@@ -61,18 +62,29 @@ export class allAffiliates {
   }
 
   setFilteredValues(filteredValues) {
-    this.filteredValues =  filteredValues;
+    this.filteredValues = filteredValues;
   }
 
   parsePartner(partner) {
     const parsedDate = new Date(partner["createdStamp"]);
     return {
-      "dateTimeCreated": moment(parsedDate).format('MM-D-YYYY'),
-      "firstName": partner['firstName'],
-      "lastName": partner['lastName'],
-      "email": `${partner['firstName']}@gmail.com`,
-      "status": 'active'
+      "firstName": partner['_toOne_Person']['firstName'] ? partner['_toOne_Person']['firstName'] : 'Missing',
```

Comment:
maybe safeGet function from utilities can be useful here?
`safeGet(() => obj.prop1.prop2.prop2, defaultValue)`